app/src/main/java/io/horizontalsystems/bankwallet/modules/ratechart/CoinInfoMap.kt

```diff
@@ -119,10 +119,38 @@ object CoinInfoMap {
             "WRX"   to CoinInfo(BigDecimal(995_833_334), null, "https://wazirx.com"),
             "COS"   to CoinInfo(BigDecimal(9_400_000_000), null, "https://www.contentos.io"),
             "LTO"   to CoinInfo(BigDecimal(500_000_000), null, "https://lto.network"),
-            "MTXLT" to CoinInfo(BigDecimal(900_000), null, "https://tixl.me")
-
-
-    )
+            "MTXLT" to CoinInfo(BigDecimal(900_000), null, "https://tixl.me"),
+            "GRT" to CoinInfo(BigDecimal(10_000_000_000), "15/12/2020", "https://thegraph.com/"),
```

Comment:
Can you remove last backslash `/` from URL's. This way it will look prettier in app.